cmd/device-worker/main.go

```diff
@@ -162,17 +160,30 @@ func main() {
 		metricsStore,
 	)
 
+	dataDirPlaybook := path.Join(baseDataDir, "devicePlaybooks")
+	if err := os.MkdirAll(dataDirPlaybook, 0750); err != nil {
+		log.Fatalf("cannot create directory: %v", err)
+	}
+	ansibleManager, err := ansible.NewAnsibleManager(dispatcherClient, dataDirPlaybook)
+	if err != nil {
+		log.Fatalf("cannot start ansible manager, err: %v", err)
```

Comment:
Do we consider Ansible as hard dependency, so without Ansible we can't use normal workloads? Or should it be soft, so if Ansible is not installed, we still can use device-worker, but without Ansible the Playbooks CRs will fail to execute?

Reply:
I added this because when there wasn't `ansible` in gitlab ci the test in my PR panicked (https://github.com/project-flotta/flotta-device-worker/runs/5557779128?check_suite_focus=true#step:5:18). 
I see your point. Let me fix it.